dialogue-serde/src/main/java/com/palantir/conjure/java/dialogue/serde/DefaultClients.java

```diff
@@ -120,4 +126,20 @@ public void onFailure(Throwable throwable) {
             log.info("Canceled call failed", throwable);
         }
     }
+
+    private static Request accepting(Request original, String acceptValue) {
+        Preconditions.checkNotNull(acceptValue, "Accept value is required");
+        Preconditions.checkState(!acceptValue.isEmpty(), "Accept value must not be empty");
+        if (original.headerParams().containsKey(HttpHeaders.ACCEPT)) {
```

Comment:
should we append the acceptValue?

Reply:
I don't think so, we'd need to deduplicate repeated accept values, and the existing values will be preferred since we're using a list, which isn't what we want here. I considered throwing instead of logging but it's difficult to trace where the value was added (unlike the null/empty inputs) so we would want to return a future.